src/messages/MMonMap.h

```diff
@@ -24,10 +24,17 @@
 class MMonMap final : public Message {
 public:
   ceph::buffer::list monmapbl;
+  entity_addrvec_t peer_addrs;
+  std::set<int> quorum;
+  epoch_t epoch = 0;
```

Comment:
Is this duplicating `MonMap::epoch`? Why?

Reply:
Hmm, this might be a sign we're de facto extending the `MonMap` but in clumsy way.